src/condor_contrib/bosco/bosco_cluster

```diff
@@ -839,10 +841,13 @@ fi
 start_ssh
 
 # Quickly test the ssh
-qmgr_out=`ssh -o "PasswordAuthentication=no" $remote_host "pwd" 2>/dev/null`
+ssh_opts='-o "PasswordAuthentication=no"'
+[ $debug -eq 0 ] || ssh_opts="$ssh_opts -vvvvv"
```

Comment:
```suggestion
[[ $debug = 1 ]] && ssh_opts="$ssh_opts -vvv"
```

If you did not set a default value for `$debug` anywhere (i don't see a default `debug=0`), then this expression `[ $debug -eq 0 ]` will expand as `[ -eq 0 ]` which is not a valid test expression and will not behave like you want.  Really `-eq` is meant to be used only when both arguments are expected to be numeric (int) values.

You can fix this by setting a default `default=0` above the getopt command line parsing.

You _could_ also change it to `[[ $debug -eq 0 ]]`, which is a little non-obvious but it works because it treats an unset `$debug` as `""` which in the numeric context of `-eq` evaluates to `0`.

Another more straightforward & obvious fix is just `[[ $debug = 1 ]] && ...`  (which works here although in general i am in favor of `||` expressions over `&&`, as the latter can cause unexpected gotchas if it appears in the final statement of a function or script).

Oh, and, re `-vvvvv`:

```
  -v    Verbose mode. ...
        Multiple -v options increase the verbosity.
        *The maximum is 3.*
```